nchelpers/util.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
```

Comment:
Is there enough code in here to warrant a util module? If so, maybe we could rename it to something more descriptive (`dateutil`?). I mean, don't get me wrong... I'm equally guilty of naming modules `util`. But then I hate myself for it thereafter every time I have to touch the code. :)